capnpc-go/templates/interfaceClient

```diff
@@ -30,3 +30,11 @@ func (c {{$.Node.Name}}) AddRef() {{$.Node.Name}} {
 func (c {{$.Node.Name}}) Release() {
 	c.Client.Release()
 }
+
+func (c {{$.Node.Name}}) EncodeAsPtr(seg *{{.G.Capnp}}.Segment) {{.G.Capnp}}.Ptr {
```

Comment:
Mentioned in the commit message, but: these generated wrappers are necessary for capabilities but not for structs, I think because the generated struct types *embed* `capnp.Struct`, whereas the interface types have an explicitly named field called `Client`. I don't seen an obvious reason why these should be different, so maybe we should just change this so that `Client` is embedded as well? I'm inclined to suggest that as a separate PR though, to keep things orthogonal.

Reply:
SGTM.  Since we look at codegen relatively rarely, I'm going to open your comment as a new issue so that we don't forget.